packages/lesswrong/server/rateLimitUtils.ts

```diff
@@ -27,6 +27,7 @@ async function getPostsInTimeframe(user: DbUser, maxHours: number) {
   return await Posts.find({
     userId:user._id, 
     draft: false,
+    event: false, // I've never seen event-spam, and sometimes people need to make 100+ events for special occassions -- Ray
```

Comment:
I think this should be `isEvent`, actually

Reply:
Looks like this is the cause of the Cypress failure too